src/wp-includes/rest-api/endpoints/class-wp-rest-autosaves-controller.php

```diff
@@ -372,6 +374,20 @@ public function create_post_autosave( $post_data ) {
 			}
 		}
 
+		// Check if meta values have changed.
+		if ( ! empty( $meta ) ) {
+			$revisioned_meta_keys = wp_post_revision_meta_keys( $post->post_type );
+			foreach ( $revisioned_meta_keys as $meta_key ) {
+				$old_meta = get_post_meta( $post_id, $meta_key, true );
```

Comment:
What if there is a default value is set for this key? Should we be using `get_metadata_raw` here?

Reply:
good question, I'm not sure what the behavior should be for meta with default values. I will think on this further, and we should certainly add tests here to verify it behaves as we expect.

Reply:
Good catch, changed.

Reply:
Going to check this manually before pushing since our unit tests don't really cover this.

Reply:
Updated this in [`e1950b0` (#4859)](https://github.com/WordPress/wordpress-develop/pull/4859/commits/e1950b0debef4eeb629f932a9dbe0dbe81cd6a16)